app/src/filters/components/FilterGroupForm.jsx

```diff
@@ -62,6 +65,63 @@ class FilterGroupForm extends Component {
   onFilterValueChange(name, value) {
     const filterGroup = this.state.filterGroup;
     filterGroup.filterValues[name] = value;
+
+    this.setState({ filterGroup });
+
+    if (this.state.autoGenLabel) {
+      this.genFilterName();
+    }
+  }
+
+  genFilterName() {
```

Comment:
@tiagojsag Not sure about this: At first is useful but if you change any filter it doesn't update and it could be confusing for the user if he/she forgets to change it, also if we have many filters or the labels are long. We may want to check this with Ariadna.

Reply:
Seems logical to me